stable/database/templates/statefulset.yaml

```diff
@@ -608,3 +608,185 @@ spec:
   {{- end }}
 {{- end }}
 {{- end }}
+
+{{- if .Values.database.te.enableStatefulSet }}
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: {{ template "database.fullname" . }}
+    group: nuodb
+    database: {{ .Values.database.name }}
+    domain: {{ .Values.admin.domain }}
+    chart: {{ template "database.chart" . }}
+    release: {{ .Release.Name | quote }}
+  name: te-{{ template "database.fullname" . }}
+spec:
+  serviceName: {{ .Values.database.name }}
```

Comment:
2) Specify service name.

Maybe this should be conditionalized on `.Values.database.te.dbServices.enabled`.